### ISSUE TITLE
add GBIF DOIs to Zenodo Parasite Tracker pub 

### ISSUE THREAD

**Issue by jhpoelen** [ParasiteTracker/tpt-reporting#11]
Zenodo provides a way to reference datasets that were used to make a data publication.

GBIF issues DOIs to help cite metadata for a GBIF registered dataset.

Some Parasite Tracker datasets are registered with GBIF .

Suggest to add GBIF DOIs to Zenodo parasite tracker publication as a way to cite the datasets used in the TPT report at https://zenodo.org/record/3685365 .

**Comment by jhpoelen** [comment]
@mdoering @timrobertson100 @MattBlissett does GBIF doi tracker pick up citations from Zenodo pubs?

**Comment by timrobertson100** [comment]
I’m not sure how automatic it’d be but I don’t see why it shouldn’t be linked. Here are similar examples if I understand correctly:

https://www.gbif.org/resource/search?q=Zenodo&contentType=literature&literatureType=generic

**Comment by jhpoelen** [comment]
@timrobertson100 thanks for the example! Great to see that the datasets referenced in the Zenodo pub make it all the way to GBIF usage tracking!

Just for the record: from your search link (see attached screenshot), I found https://doi.org/10.5281/zenodo.3632750 and in that publication I found "derived from" links (see attached screenshot) to DOIs of GBIF download query pages https://doi.org/10.15468/dl.g1z7y7 and https://doi.org/10.15468/dl.db9azb . I am assuming the same would happen when GBIF dataset landing page DOIs are referenced.

![Screenshot from 2020-03-02 12-50-16](https://user-images.githubusercontent.com/1084872/75716769-b4fadf80-5c84-11ea-959b-612bf39f259f.png)
![Screenshot from 2020-03-02 12-49-11](https://user-images.githubusercontent.com/1084872/75716778-b88e6680-5c84-11ea-89c3-13eb476d59f2.png)

**Comment by dnoesgaard** [comment]
> @mdoering @timrobertson100 @MattBlissett does GBIF doi tracker pick up citations from Zenodo pubs?

Yes and no. Technically speaking, these citations should make it into Event Data, which we're using as a source for linking literature (and datasets) to GBIF data. However, it has been very inconsistent and not fit as a primary source. So while we might add these links eventually, it's more likely to be based on a catch from, say Google Scholar or similar.

**Comment by jhpoelen** [comment]
@dnoesgaard thanks for chiming in. Is there another method to nudge GBIF's usage tracking method to pickup GBIF dataset page DOI references from Zenodo (or similar) data pubs?

**Comment by dnoesgaard** [comment]
@jhpoelen Links between Zenodo resources (with Datacite DOIs) should end up in Event data and I'll keep exploring that. But if you have specific examples, I'd love to have them. If nothing else, I'll be able to add them manually.

**Comment by jhpoelen** [comment]
@dnoesgaard ok - will provide examples when we have them. Thanks for engaging.
